tox.ini

```diff
@@ -7,16 +7,17 @@ testdeps =
 [tox]
 envlist =
     py{37,38,39,310}
-    py38-lint
-    py38-coverage
-    py38-docs
+    lint
+    coverage
+    docs
 
 [testenv]
 deps = {[base]testdeps}
 extras = plotly
 commands = pytest {posargs}
 
-[testenv:py38-lint]
+[testenv:lint]
+basepython=python3.8
```

Comment:
I think it would be better to use py37 for linting, so we are sure it will catch everything (if we use a py38 feature it will not complain but it will fail with py37) and the CI will be faster since all the jobs will not execute in the same environment.

Reply:
I thought about that but one would need most interpreters to run tox locally.

Reply:
It's easy to install ;-)

Reply:
Not on mac :P

Reply:
Nah, you can install all of them, it is just they are not on your path out-of-the-box. I will change lint to py37 and I will deal with the mac side.

Reply:
Done

Reply:
Cool, thanks!